### PR TITLE
Don’t add blocked users to new groups and handle block alert results …

### PR DESCRIPTION
**Pull request by charlesmchen**
…correctly.

PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/ViewControllers/NewGroupViewController.m [hunk 1]
```diff
@@ -205,37 +206,56 @@ - (void)updateTableContents
...
-                    // In the "contacts" section, we label members as such when editing an existing group.
-                    cell.accessoryMessage = NSLocalizedString(
-                        @"NEW_GROUP_MEMBER_LABEL", @"An indicator that a user is a member of the new group.");
-                } else if (isBlocked) {
-                    cell.accessoryMessage = NSLocalizedString(
-                        @"CONTACT_CELL_IS_BLOCKED", @"An indicator that a contact has been blocked.");
-                } else {
-                    OWSAssert(cell.accessoryMessage == nil);
-                }
+            [nonContactsSection
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/NewGroupViewController.m]: looks like the `customCellBlock` part is just clang-format having a party?

File: Signal/src/ViewControllers/NewGroupViewController.m [hunk 2]
```diff
@@ -205,37 +206,56 @@ - (void)updateTableContents
...
-                return cell;
-            }
-                                            customRowHeight:[ContactTableViewCell rowHeight]
-                                            actionBlock:^{
-                                                [weakSelf removeRecipientId:recipientId];
-                                            }]];
+                    return cell;
+                }
+                            customRowHeight:[ContactTableViewCell rowHeight]
+                            actionBlock:^{
```
Comment (michaelkirk) [inline Signal/src/ViewControllers/NewGroupViewController.m]: ...and that the substantive change is here. (this part LGTM)

File: Signal/src/ViewControllers/NewGroupViewController.m (hunk 1 above)
Comment (charlesmchen) [inline Signal/src/ViewControllers/NewGroupViewController.m]: Yeah ¯\_(ツ)_/¯

Comment (charlesmchen) [inline Signal/src/ViewControllers/NewGroupViewController.m]: I'm inclined to let clang-format have its way.

**Pull request merged**